src/workloads/networking/CommitLatencySingleUpdate.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Single threaded updates to measure commit latency for various write concern.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+Actors:
+  - Name: SingleThreadUpdate
+    Type: RunCommand
+    Threads: 1
+    Database: test
+    Phases:
+      - Repeat: 500
+        Operations:
+        - OperationCommand:
+            update: accounts
+            updates: [ { q: { _id: 1 }, u: { $inc: { n: 1 } } } ]
```

Comment:
Should there be an insert somewhere before these updates?

Reply:
Note: While adding this I also learned that RunCommand exits on error, such as when dropping a non-existing collection. So I have also refactored CommitLatency.yml so that drop and insert are 2 distinct phases.